mono/mini/driver.c

```diff
@@ -200,9 +200,7 @@ static gboolean
 parse_debug_options (const char* p)
 {
 	MonoDebugOptions *opt = mini_get_debug_options ();
-#ifdef ENABLE_NETCORE
 	opt->enabled = TRUE;
-#endif
```

Comment:
this doesn't look correct, it'd enable debug symbol reading by default in non-netcore mono too.

Reply:
It is what was there.
It was only changed as a guess to fix this bug.
I believe it is correct, because reads are iifdef'ed.

Reply:
yeah but that's just a side-effect of the implementation because non-netcore mono doesn't respect this flag.

We should either add the ifdef back or remove the ifdef everywhere and just default to `FALSE` on non-netcore mono.

Reply:
Ifded should be minimized, esp. in data structures and headers. I’ll add a global constant for netcore and then remove the remaining ifdef. Meanwhile *seems* to be working.